webpack.config.base.cjs

```diff
@@ -0,0 +1,24 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
I would move this file to webpack package, to have everything related to webpack in one place